src/pages/about/index.tsx

```diff
@@ -0,0 +1,139 @@
+import {
+  Grid,
+  Heading,
+  View,
+  Text,
+  Button,
+  useBreakpointValue,
+} from "@aws-amplify/ui-react";
+import { GetStaticPropsContext, GetStaticPropsResult } from "next";
+import { ActionLayout } from "../../components/ActionLayout";
+import { ContributorCollection } from "../../components/ContributorCollection";
+import { Layout } from "../../components/Layout";
+import ExternalIconCustom from "../../ui-components/ExternalIconCustom";
+import { serializeModel, deserializeModel } from "@aws-amplify/datastore/ssr";
+import { trackExternalLink } from "../../utils/track";
+import { Contributor } from "../../models";
+import { Context } from "../../types/models";
+import { getContributors } from "../../lib/getData";
+
+export default function AboutPage(data: any) {
+  const contributors: Contributor[] = deserializeModel(
+    Contributor,
+    data.contributors
+  );
+
+  const useLargeVariant = useBreakpointValue({
+    base: false,
+    small: false,
+    medium: false,
+    large: false,
+    xl: true,
+  }) as boolean;
+
+  const collectionType = useBreakpointValue({
+    base: "list",
+    small: "list",
+    medium: "grid",
+    large: "grid",
+  }) as "grid" | "list";
+
+  return (
+    <Layout>
+      <View columnStart="2">
+        <Grid
+          templateColumns={{
+            base: "1fr",
+            small: "1fr",
+            medium: "1fr",
+            large: "1fr 1fr",
+          }}
+          gap="16px"
+          marginBottom={{
+            base: "64px",
+            small: "64px",
+            medium: "64px",
+            large: "128px",
+          }}
+        >
+          <Heading fontFamily="Amazon Ember" fontWeight="300" level={1}>
+            About
+          </Heading>
+          <Text columnStart="1">
+            This site contains a collection of courses on{" "}
+            <a
+              target="_blank"
+              href="https://docs.amplify.aws"
+              rel="noopener noreferrer"
+            >
+              AWS Amplify
+            </a>{" "}
+            brought to you by the Developer Advocacy team working on the
+            product.
+          </Text>
+        </Grid>
+        <ContributorCollection
+          contributors={contributors}
+          type={collectionType}
+          useLargeVariant={useLargeVariant}
+          gap="20px"
+          templateColumns={{
+            base: "1fr",
+            small: "1fr",
+            medium: "1fr 1fr",
+            large: "1fr 1fr",
+            xl: "1fr 1fr 1fr",
+          }}
+        />
+        <ActionLayout>
+          <View as="div">
+            <Heading fontFamily="Amazon Ember" fontWeight="300" level={3}>
+              {`Let's Connect`}
+            </Heading>
+            <p>
+              Join the Amplify Community Discord to ask questions, contribute to
+              the open source project, and learn about new features.
+            </p>
+          </View>
+          <View as="div">
```

Comment:
`<View>` renders a `<div>` by default, so it's not necessary to use the `as` prop here